app/modes/initmodes.F90

```diff
@@ -181,6 +183,17 @@ subroutine initProgramVariables()
     ! oscillation cycles in an animation
     nCycles = 3
 
+    ! Eigensolver
+    call getChildValue(root, "EigenSolver", buffer2, "qr")
+    select case(tolower(char(buffer2)))
+    case ("qr")
+      iSolver = 1
+    case ("divideandconquer")
+      iSolver = 2
+    case default
```

Comment:
Any reason not to add the "relativelyrobust" solver as well?

Reply:
Not really, will require a little bit of back-end work, as the RR solver is only partly wrapped (the generalized case being assembled on top and then exposed).

Reply:
@vanderhe added